geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/ShowMetricsDUnitTest.java

```diff
@@ -51,9 +40,15 @@
 import org.apache.geode.test.dunit.SerializableCallable;
 import org.apache.geode.test.dunit.SerializableRunnable;
 import org.apache.geode.test.dunit.VM;
-import org.apache.geode.test.dunit.WaitCriterion;
 import org.apache.geode.test.junit.categories.DistributedTest;
 import org.apache.geode.test.junit.categories.FlakyTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Properties;
+import javax.management.ObjectName;
 
 @Category({DistributedTest.class, FlakyTest.class}) // GEODE-1764 GEODE-3530
 @SuppressWarnings("serial")
```

Comment:
Maybe now it's a good time to rewrite this test into using the rules. I can pair with you on this one.